### ISSUE TITLE
Bug: Errors in WebXR Samples - Controller State

### ISSUE THREAD

**Issue by De-Panther** [meta-quest/immersive-web-emulator#34]
When using the Controller State sample
https://immersive-web.github.io/webxr-samples/controller-state.html
First there's an error
```
motion-controllers.module.js:120 Uncaught (in promise) Error: No matching handedness, , in profile generic-hand
    at fetchProfile (motion-controllers.module.js:120:13)
fetchProfile @ motion-controllers.module.js:120
Promise.then (async)
(anonymous) @ input-renderer.js:248
dispatchEvent @ webxr-polyfill.js:2958
_checkInputSourcesChange @ webxr-polyfill.js:3391
XRSession$1.<computed>.onDeviceFrame @ webxr-polyfill.js:3161
requestAnimationFrame (async)
requestAnimationFrame @ webxr-polyfill.js:27152
XRSession$1.<computed>.startDeviceFrameLoop @ webxr-polyfill.js:3178
XRSession$1 @ webxr-polyfill.js:3252
requestSession @ webxr-polyfill.js:4461
await in requestSession (async)
CustomWebXRPolyfill.XRSystem.requestSession @ webxr-polyfill.js:32464
initXR @ controller-state.html:254
(anonymous) @ controller-state.html:364
```
![image](https://github.com/meta-quest/immersive-web-emulator/assets/4749703/6f9ed709-926e-4f43-bfbb-3078b535a9e2)

And when entering the XR session, the hands poses are tracked, but not the joints or pinch input.

**Comment by felixtrz** [comment]
The hand in this sample is not animated on device or through IWE. There is no implementation to animate the hand mesh in the sample's source code. But something is definitely wrong with buttons exposed through emulated hand tracking, I will fix it.

**Comment by De-Panther** [comment]
IWE?

I think that the first issue, is that while in `inline` session, the emulator is trying to run other sessions operations, like `_checkInputSourcesChange`.
And I guess that once there's an error, other stuff stops working as intended.

**Comment by felixtrz** [comment]
It's indeed curious.. it seems to be a bug in the WebXR polyfill library that the emulator is using, I will patch that up.

IWE == Immersive Web Emulator

**Comment by felixtrz** [comment]
The fix has been landed into the main branch. If you'd like immediate access, you can build from the source to obtain the patch. Otherwise, it will be included in the upcoming **v1.4.2** release.

Thank you for bringing this issue to our attention.
